apps/sms/views/shared/test/unit/localization_helper_test.js

```diff
@@ -52,11 +46,16 @@ suite('LocalizationHelper >', function() {
     customIframeDocument.documentElement.lang = 'en-US';
     customIframeDocument.documentElement.dir = 'ltr';
 
-    navigator.mozL10n.language.code = 'be-BY';
-    navigator.mozL10n.language.direction = 'rtl';
-    navigator.mozL10n.ready.yield();
+    LocalizationHelper.init().then(
+      () => undefined).then(done, done);
```

Comment:
I needed this to make sure `setup` waits for the localization of iframes.  This approach required small changes to the `LocalizationHelper` so that `init` returns a promise.

An alternative approach would be to do something like:

    LocalizationHelper.init();
    Promise.resolve().then(done, done);

The way microtasks are scheduled, this achieves the same goal, but perhaps is more cryptic. I guess it a trade-off for not having to make `init` return a promise.

@julienw, what's your preference?